File: ERAS.Server/Models/EquipmentModel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ERAS.Server.Models
{
    public class EquipmentModel
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Alias { get; set; }
        public bool? IsDeleted { get; set; }
        public DateTimeOffset? DateCreated { get; set; }
        public DateTimeOffset? DateModified { get; set; }
        public int? UserCreated { get; set; }
        public int? UserModified { get; set; }
    }
}

File: eras.client/src/app/models/equipment-model.ts
import { User } from '../models/user';
export interface EquipmentModel {
  id: number,
  name?: string,
  alias?: string,
  isDeleted?: boolean,
  dateCreated?: Date,
  dateModified?: Date,
  userCreatedId?: number,
  userModifiedId?: number,
  userCreated?: User;
  userModified?: User;
}
